.github/pull_request_template.md

```diff
@@ -0,0 +1,5 @@
+Complete each step to get your pull request merged in. [Learn more about the workflow this project uses](https://github.com/customerio/customerio-ios/develop/docs/dev-notes/GIT-WORKFLOW.md). 
+- [ ] [Assign members of your team](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/requesting-a-pull-request-review) to review the pull request. 
+- [ ] Wait for pull request status checks to complete. If there are problems, fix them until you see that [all status checks are passing](https://external-content.duckduckgo.com/iu/?u=https%3A%2F%2Fsymfony.com%2Fdoc%2F4.3%2F_images%2Fdocs-pull-request-symfonycloud.png&f=1&nofb=1). 
+- [ ] Wait until the pull request has been reviewed *and approved* by a teammate
+- [ ] After pull request is approved, and you determine it's ready **add the label "Ready to merge"** to the pull request and it will automatically be merged. 
```

Comment:
When will it be merged ? Immediately or after a certain period of time ?

Reply:
Immediately. A bot will merge the request for you. 

You know how today when we squash merge pull requests, we need to remember to delete the github autogenerated description? And sometimes github has made the squashed commit *not* the title of the pull request like we want? By having a bot merge our PR, the bot will do all of this for us so we don't need to remember to do anything!

Reply:
I like this auto-botic merging. Can you mention that `it will automatically be merged as soon as you add the tag`
When will it be merged ? This question popped up in my mind as I read. I feel other devs might also feel the same.

Reply:
Thanks for bringing that up. 

I added a note about a bot merging the PR. Does the change look good to you?

I wanted to avoid using the word "immediately" because it's not immediate but more like 15 seconds until the bot merges it.

Reply:
This looks perfect 👍